src/Liquid.Repository.Mongo/Exceptions/MongoDatabaseDoesNotExistException.cs

```diff
@@ -12,12 +13,27 @@ namespace Liquid.Repository.Mongo.Exceptions
     [ExcludeFromCodeCoverage]
     public class MongoDatabaseDoesNotExistException : LightException
```

Comment:
Shouldn't this inherit from DatabaseContextException ? Also, since both implementations (Mongo and EF) share a similar exception (DB Not Found), shouldn't there be a common exception?